src/world_interface/base_motor.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+#include "data.h"
+#include "../control/JacobianVelController.h"
+#include "../utils/scheduler.h"
+
+using namespace std::chrono_literals;
```

Comment:
We should avoid `using` directives in header files. Seems like this isn't used anywhere in this file anyway, so can you just remove it?